plugins/frontend/backstage-plugin-iframe/src/components/IFrameComponent.tsx

```diff
@@ -28,12 +30,20 @@ import { determineError } from './utils/helpers';
  */
 export const IFrameCard = (props: IFrameProps) => {
   const { src, height, width } = props;
+  const { entity } = useEntity();
+
   const title =
     props.title || 'Backstage IFrame (Note you can modify this with the props)';
   const configApi = useApi(configApiRef);
   const allowList = configApi.getOptionalStringArray('iframe.allowList');
   const errorMessage = determineError(src, allowList);
 
+  // In theory this might be extended to include the logged in user maybe.
+  const evaluatedSrc = renderString(src, { entity });
```

Comment:
This is unsanitised user input right? I'm not sure how big an issue that is.. 

<img width="608" alt="Screenshot 2022-08-10 at 16 35 27" src="https://user-images.githubusercontent.com/1415599/183946078-3164b5c6-ca89-49fc-a9e0-cb66c3368c13.png">
<img width="948" alt="Screenshot 2022-08-10 at 16 36 32" src="https://user-images.githubusercontent.com/1415599/183946348-235b3f0f-354f-4889-bea7-8d7b496c86c0.png">

Reply:
I am attempting to sanitise the url a little below this line. I am not exactly sure if it is enough.